bpython/args.py

```diff
@@ -152,9 +149,7 @@ def callback(group):
         help=_("Set log level for logging"),
     )
     parser.add_argument(
-        "--log-output",
-        "-L",
-        help=_("Log output file"),
+        "--log-output", "-L", help=_("Log output file"),
```

Comment:
this looks like another bad reformat, I won't comment on any others but these formatting-only changes shouldn't be in this PR.